src/configs/index.js

```diff
@@ -81,3 +126,105 @@ export const formRegExp = {
   deliveryType: "^(?=.{1,30}$)[a-zA-Z]+(([',. -][a-zA-Z ])?[a-zA-Z]*)*$",
   deliveryMethod: "^(?=.{1,30}$)[a-zA-Z]+(([',. -][a-zA-Z ])?[a-zA-Z]*)*$"
 };
+
+export const placeholders = {
```

Comment:
Please, create a file with translations inside the `translations` folder, this code should not be here, in config file

Reply:
moved